src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -88,7 +91,36 @@ class F extends ValidationTest {
 
 export const g = makeTestGroup(F);
 
-g.test('it_is_invalid_to_draw_in_a_render_pass_with_missing_bind_groups')
+g.test('set_pipeline_or_bind_group_missing_in_render_pass')
```

Comment:
It isn't necessary to test what happens if the pipeline is missing when you issue a draw call; that's not part of pipeline_bind_group_compat rules and I think we already have tests for it.

Reply:
Removed.